src/arkhamdb/DeckImporterMain.ttslua

```diff
@@ -430,6 +432,22 @@ function loadCards(slots, investigatorId, customizations, playerColor, commandMa
   startLuaCoroutine(self, "coinside")
 end
 
+-- Callback handler for the main deck spawning.  Looks for cards which should start in hand, and
+-- draws them for the appropriate player.
+-- @param deck Callback-provided spawned deck object
+-- @param playerColor Color of the player to draw the cards to
+function deckSpawned(deck, playerColor)
+  local handPos = Player[playerColor].getHandTransform(1).position -- Only one hand zone per player
+  local deckCards = deck.getData().ContainedObjects
+  -- Process in reverse order so taking cards out doesn't upset the indexing
+  for i = #deckCards, 1, -1 do
+    local cardMetadata = JSON.decode(deckCards[i].GMNotes) or { }
+    if cardMetadata.startsInHand then
+      deck.takeObject({ index = i - 1, position = handPos, flip = true, smooth = true})
```

Comment:
I assume cards with "startsInHand" are always at the top of the deck?

Reply:
No, they'll be in the usual order.  We'd have to parse the deck anyway, simpler to just loop the whole thing and not try to add special casing or be dependent on the ordering happening elsewhere.